README.md

```diff
@@ -4,7 +4,7 @@
 [![](https://img.shields.io/static/v1?style=social&label=Donate&message=%E2%9D%A4&logo=Paypal&color&link=%3Curl%3E)](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=ZP5F38L4C88UY&source=url)
 [![](https://img.shields.io/twitter/follow/hediet_dev.svg?style=social)](https://twitter.com/intent/follow?screen_name=hediet_dev)
 
-This unofficial extension integrates [Draw.io](https://app.diagrams.net/) (also known as [diagrams.net](diagrams.net)) into VS Code.  
+This unofficial extension integrates [Draw.io](https://app.diagrams.net/) (also known as [diagrams.net](https://www.diagrams.net/)) into VS Code.  
```

Comment:
As far as I know there are several ways as follows:

![image](https://github.com/hediet/vscode-drawio/assets/124909303/d06bbfac-eb81-423c-826f-9b6853a8d642)

Reply:
@huyndev24 yes it is valid markdown,
but on GiHub this is interpreted as a relative path to a file named `diagrams.net` which doesn't exist.
To reproduce this you can click in the link in the repo Main page:
![Screenshot_20230804-182358.png](https://github.com/hediet/vscode-drawio/assets/11715448/933341e4-0fd9-4ac9-a001-87446cf01321)